src/commands/music/play.ts

```diff
@@ -14,14 +16,14 @@ module.exports = {
             .setName( 'station' )
             .setDescription( 'Choose the radio station' )
             .setRequired( true )),
-	async execute( client, interaction ) {
+	async execute( client, interaction, switchStation ) {
 
-        const station = interaction.options.getString( 'station' ).toLowerCase()
+        const station = interaction.options?.getString( 'station' )?.toLowerCase() || switchStation.toLowerCase()
 
-        if (! fs.existsSync( `../music/${station}`) ) {
+        if (! fs.existsSync( `music/${station}`) ) {
             return await interaction.reply( 'I could not find that radio station, are you sure it exists?' )
         }
-        fs.readdirSync( `../music/${station}`, async ( err, files ) => {
+        fs.readdirSync( `music/${station}`, async ( err, files ) => {
```

Comment:
This requires the music folder to be in the current working directory, which is different from the previous behavior. Maybe we should make the music folder location a part of the configuration file, instead of hard-coding it?

Reply:
Actually the previous path was not at the root, hence the commands did not work. Hence I changed it to be the root. I too feel that having it in the configuration file would indeed be better. Should I make this change?

Reply:
It did work for me with the music folder in the root dir so I don't know why it didn't work, allowing the user to specify the location of the file could be added but I think it only makes things more complicated so I don't know how much it will add.